packages/select/src/SelectTheme.ts

```diff
@@ -43,7 +43,9 @@ export interface SelectTheme {
     hoverBackgroundColor: string;
     minWidth?: string;
     selectedItemTextColor: string;
+    selectedItemHoverTextColor: string;
     selectedItemBackgroundColor: string;
+    selectedItemHoverBackgroundColor: string;
```

Comment:
state at end of variable name :)

Reply:
The entire select theme is messed up in that case.